azure/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@microsoft/vscode-azext-azureutils",
     "author": "Microsoft Corporation",
-    "version": "0.3.8",
+    "version": "1.0.0",
```

Comment:
<img src="https://media1.giphy.com/media/ie4fEHT4krdDO/giphy.gif"/>

Reply:
lol I thought we agreed to this?!?!

Reply:
The gif was supposed to relay celebration, but looks like I picked the wrong gif. I really wanted a Kramer gif. I was lazy and didn't take the time to pick the right gif. That's on me.

Reply:
<img src="https://media2.giphy.com/media/l0MYMiMPgXO3u6UZa/giphy.gif"/>